acceptancetests/deploy_stack.py

```diff
@@ -736,6 +777,27 @@ def from_args(cls, args):
         return cls.from_client(args, client)
 
     @classmethod
+    def from_existing(cls, args):
+        juju_home = os.environ['JUJU_HOME']
```

Comment:
It may be useful to provide a more descriptive message here if JUJU_HOME isn't set. If you can catch the KeyError and explain the need to set juju home when using existing, it may smooth over onboarding. I suspect developers / test writers to use this, and thus a little hand holding here may help.